### ISSUE TITLE
Add missing exchange arrows in Europe

### ISSUE THREAD

**Issue by alixunderplatz** [tmrowco/electricitymap#1197]
Even though there is no data at the moment for some of the zones, I think it's nice to complete the cross-border exchanges with data from ENTSO-E, that are not yet displayed.
Maybe values were simply not available in the past, so consider this a list of possible updates for exchanges :)

1.) [BE-LU](https://transparency.entsoe.eu/transmission-domain/physicalFlow/show?name=&defaultValue=false&viewType=TABLE&areaType=BORDER_CTY&atch=false&dateTime.dateTime=12.03.2018+00:00|CET|DAY&border.values=CTY|10YLU-CEGEDEL-NQ!CTY_CTY|10YLU-CEGEDEL-NQ_CTY_CTY|10YBE----------2&dateTime.timezone=CET_CEST&dateTime.timezone_input=CET+(UTC+1)+/+CEST+(UTC+2))
2.) [DE-LU](https://transparency.entsoe.eu/transmission-domain/physicalFlow/show?name=&defaultValue=false&viewType=TABLE&areaType=BORDER_CTY&atch=false&dateTime.dateTime=12.03.2018+00:00|CET|DAY&border.values=CTY|10YLU-CEGEDEL-NQ!CTY_CTY|10YLU-CEGEDEL-NQ_CTY_CTY|10Y1001A1001A83F&dateTime.timezone=CET_CEST&dateTime.timezone_input=CET+(UTC+1)+/+CEST+(UTC+2))
3.) [UA-BY](https://transparency.entsoe.eu/transmission-domain/physicalFlow/show?name=&defaultValue=false&viewType=TABLE&areaType=BORDER_CTY&atch=false&dateTime.dateTime=12.03.2018+00:00|CET|DAY&border.values=CTY|10Y1001C--00003F!CTY_CTY|10Y1001C--00003F_CTY_CTY|BY&dateTime.timezone=CET_CEST&dateTime.timezone_input=CET+(UTC+1)+/+CEST+(UTC+2))
4.) IT-MT (exchange data is there if you select Malta on the electricitymap, just the arrow must be added)

**Comment by corradio** [comment]
Good initiative! However I think that the LU arrows will never be relevant because LU is part of another bidding zone?

**Comment by alixunderplatz** [comment]
@corradio LU is part of the bidding zone DE-AT-LU. This only means, the electricity price in these three countries is always the same. However, exchanges between the _countries_ are given seperately.
Seems like there just is absolutely no generation data for Luxembourg at the moment on ENTSO-E. I just had a look at the generation data for the bidding zone DE-AT-LU, which simply is the sum of data for countries DE+AT.

Maybe it's worth contacting the [TSO Creos](http://www.creos-net.lu/start.html) and ask them for live data.
PV and wind energy have significant shares in installed capacities [See CREOS' "Scenario Report 2040" (p. 30 and following)](http://www.creos-net.lu/fileadmin/dokumente/NEWS/pdf/2018/Scenario_Report_2040_2_2_2018.pdf). But generation in general only has a small share to cover domestic demand. Most is covered by imports from DE and BE.

![image](https://user-images.githubusercontent.com/25743609/37283967-291d87aa-25fa-11e8-8eda-8f5a716738bb.png)


![image](https://user-images.githubusercontent.com/25743609/37284078-853b4810-25fa-11e8-91aa-42052bbc860d.png)

![image](https://user-images.githubusercontent.com/25743609/37284141-bac197e6-25fa-11e8-83c9-e2917d6fca28.png)

**Comment by corradio** [comment]
Okay noted. Thanks for looking into that!

**Comment by corradio** [comment]
Seems like the exchanges.json simply needs to be modified. Anyone wants to take a look at this?

**Comment by lorrieq** [comment]
I'll look at it

**Comment by lorrieq** [comment]
How do I get the new entries I add to exchanges.json to show up when running locally?

**Comment by corradio** [comment]
https://github.com/tmrowco/electricitymap#testing-parsers-locally

**Comment by lorrieq** [comment]
Ah of course, an exchange needs to have state data to show up and not merely be in exchanges.json.

Pull request submitted.

PS: For BY->UA (I added), the parser was only pulling back data from 15/03 even though data is available on the ENTSOE website from today. I also noticed this was happening with SK->UA (I did not add). Have you noticed this 'delay' happening with exchange data before?

PPS: Happy St Patrick's Day!
